### ISSUE TITLE
problem with the downloading data

### ISSUE THREAD

**Issue by yuzehui1996** [TrinhQuocNguyen/Edited_PConv-Keras#1]
hey,I also reproduct this project recently and I find a problem with your code(by the way, I run the  code on the server):

when I run this code in step 3:
PConvUnet(512,512).summary()

it downloading data from  https://github.com/fchollet/deep-learning-models/releases/download/v0.1/vgg16_weights_tf_dim_ordering_tf_kernels_notop.h5

but it doesn't work any more...
how to deal with the problem?
thanks~

**Comment by TrinhQuocNguyen** [comment]
Hi yuzehui1996,
Did you run with jupyter notebook?
It is supposed to download that network to extract "knowledge" what the network has learned from Imagenet dataset.

**Comment by yuzehui1996** [comment]
yes

**Comment by TrinhQuocNguyen** [comment]
Then did you run the next "code block"?? could you capture the image what you did?

**Comment by yuzehui1996** [comment]
it can show the image from my dataset path,but it only shows the mask,the masked image and the ori image.

**Comment by yuzehui1996** [comment]
by the way, I have download the vgg16_weights_tf_dim_ordering_tf_kernels_notop.h5 on my localhost，and i can upload it into the server. Is there any ways I can load the .h5 file instead of downloading?

**Comment by TrinhQuocNguyen** [comment]
In pconv_model.py file, at this line: pconv_model.pyfrom keras.applications import VGG16
keras framework will download and use the VGG16 network, if you want to upload it by hand, you have to write code for yourself.
```
    def build_vgg(self):
        """
        Load pre-trained VGG16 from keras applications
        Extract features to be used in loss function from last conv layer, see architecture at:
        https://github.com/keras-team/keras/blob/master/keras/applications/vgg16.py
        """
        # Input image to extract features from
        img = Input(shape=(self.img_rows, self.img_cols, 3))

        # Get the vgg network from Keras applications
        vgg = VGG16(weights="imagenet", include_top=False)

        # Output the first three pooling layers
        vgg.outputs = [vgg.layers[i].output for i in self.vgg_layers]

        # Create model and compile
        model = Model(inputs=img, outputs=vgg(img))
        model.trainable = False
        model.compile(loss='mse', optimizer='adam')
        
        return model
```

**Comment by TrinhQuocNguyen** [comment]
This is under construction, it is a bit messy now?　sorry :(

**Comment by yuzehui1996** [comment]
do you have any ideas about the codes? I mean the codes to upload the .h5 file by myself.

**Comment by yuzehui1996** [comment]
can you do me a favor? can you send me the .h5 file generated during the training? my email address is silver_dl@126.com.Thanks a lot!

**Comment by TrinhQuocNguyen** [comment]
Hi yuzehui1996, 
it is about 125MB, here is the link and I will delete it after 1 week, please download it.
https://drive.google.com/file/d/19qUEBCnQx0T5g05w_GkTYDJeBrljKijH/view?usp=sharing
